ledger/participant-integration-api/src/main/scala/platform/store/appendonlydao/JdbcLedgerDao.scala

```diff
@@ -208,23 +208,9 @@ private class JdbcLedgerDao(
             )
         }
 
-        val savepoint = conn.setSavepoint()
-
         val offset = validateOffsetStep(offsetStep, conn)
-        Try({
-          sequentialIndexer.store(conn, offset, Some(update))
-          PersistenceResponse.Ok
-        }).recover {
-          case NonFatal(e) if e.getMessage.contains(storageBackend.duplicateKeyError) =>
-            logger.warn(s"Ignoring duplicate configuration submission, submissionId=$submissionId")
-            conn.rollback(savepoint)
-            sequentialIndexer.store(
-              conn,
-              offset,
-              None,
-            ) // we bump the offset regardless of the fact of a duplicate
-            PersistenceResponse.Duplicate
-        }.get
```

Comment:
We have a very similar block in `storePartyEntry`. Should we remove it as well, or at least create an issue to track it?

With this PR, both the config management service and the party management service will send globally unique submission IDs to the ledger, so they should be treated the same in `JdbcLedgerDao`.

Reply:
You are absolutely right, this should be changed as well. I will create a ticket:
https://digitalasset.atlassian.net/browse/DPP-504